apps/worker/src/generate-metadata.ts

```diff
@@ -0,0 +1,9 @@
+import { PluginMetadataGenerator } from '@nestjs/cli/lib/compiler/plugins';
+
+const generator = new PluginMetadataGenerator();
+generator.generate({
```

Comment:
I think that since we don't need to have swagger here, is this file and the metadata file can be removed? Or it's still required?

Reply:
Yeah...Without adding any visitors, we won't see a metadata.ts file pop up (I found that out when Swagger was in the mix). Also, since the 'Worker' isn't using any CLI plugins, we can totally ditch that file.